docs/docsource/operation/Introduction/Conceptual_Framework.rst

```diff
@@ -1,30 +1,45 @@
 Conceptual Framework
 ====================
 
-Zenko is designed to help administrators of large storage systems
-control, manage, replicate, and visualize data in a multi-cloud context.
+Zenko is designed to help administrators of large storage systems control,
+manage, replicate, and visualize data in a multi-cloud context.
 
-Fundamental building blocks of Zenko’s replication framework are the
-bucket, the location, and the cloud service. End users (people or
-services) store data as files in buckets. Administrators are free to
-configure and name buckets as they please—to the admin, buckets are
-essentially tenants aggregated in a cloud storage deployment. From an
-end user’s perspective, a bucket simply appears as a networked storage
-location, for example, “accounting,” “bioinformatics-lab,”
-“daily-video-backup,” or any other organizational unit that makes
-sense.
+Fundamental building blocks of Zenko’s replication framework are the bucket, the
+location, and the cloud service. End users (people or services) store data as
+files in buckets. Administrators are free to configure and name buckets as they
```

Comment:
Maybe this is a nitpick of `files in buckets` but might be better as `files as objects in buckets`?

Reply:
As an end user, I store a file to a server somewhere. The rest, to the end user, is opaque. The "server" is in fact a bucket (yes?). The bucket handles the file as an object? As part of an object?

Reply:
Yeah, you're right. I think this one is fine going back over it. Marking as resolved.